content/backmatter.tex

```diff
@@ -1249,4 +1249,41 @@ \section{Version 1.1}
 %
 \end{itemize}
 
+\chapter{Errata}\label{sec:errata}
+
+The \openshmem specification may occasionally include errata that are
+discovered after the release of new versions.
+These errata can range from typographical mistakes to more significant
+technical inaccuracies that may affect the implementation or understanding of
+the \openshmem API semantics.
+This Errata section reports these issues with the goal to maintain the
```

Comment:
nit: 
section -> Section 
?

Reply:
Text should be:
"This Errata Section~\ref{sec:errata}..."